scripts/Muon/GUI/Common/corrections_tab_widget/background_corrections_model.py

```diff
@@ -43,6 +44,13 @@ def setup_functions(self):
         self.exp_decay = FunctionFactory.createFunction("ExpDecayMuon")
         self.exp_decay.setParameter("A", DEFAULT_A_VALUE)
 
+    def create_background_workspace(self) -> Workspace:
+        background = self.flat_background.getParameterValue(BACKGROUND_PARAM)
+        background_error = self.flat_background.getError(BACKGROUND_PARAM)
+        background_workspace = CreateWorkspace(DataX=[0.0], DataY=[background], DataE=[background_error],
```

Comment:
CreateSingleValuedWorkspace(OutputWorkspace='__python_binary_op_single_value', DataValue=2.5)

is from the history of https://docs.mantidproject.org/nightly/algorithms/Minus-v1.html doing the ws - 2.5 example at the end

Reply:
Thanks thats much better 👍